docs/pages/workflow/upgrading-expo-sdk-walkthrough.mdx

```diff
@@ -23,44 +23,31 @@ If you are looking to install a specific version of Expo Go, visit [expo.dev/go]
 Install the new version of the Expo package:
 
 <Tabs>
-<Tab label="npm">
-
-<Terminal
-  cmd={[
-    '# Install latest',
-    '$ npx expo install expo@latest',
-    '',
-    '# Install a specific SDK version (for example, SDK 53)',
-    '$ npx expo install expo@^53.0.0',
-  ]}
-/>
-
-</Tab>
-
-<Tab label="Yarn">
-
-<Terminal
-  cmd={[
-    '# Install latest',
-    '$ yarn expo install expo@latest',
-    '',
-    '# Install a specific SDK version (for example, SDK 53)',
-    '$ yarn expo install expo@^53.0.0',
-  ]}
-/>
-
-</Tab>
+  <Tab label="npm">
+    <Terminal cmd={['$ npm install --save expo@^53.0.0']} />
```

Comment:
Could remove `--save` as it's not needed as of npm 5 (node 8) which no one should be using.
```suggestion
    <Terminal cmd={['$ npm install expo@^53.0.0']} />
```